SU-HSDO/suhumsci
# Shs 50 twig menu poc

# READY FOR REVIEW

## Summary
This is the new mega menu V2.

## Urgency
### high

## Steps to Test
[ ] Go to the music dev site (https://music-dev.stanford.edu/). 
[ ] There is a new module there called `Stanford HumSci Mega Menu Config`. This will be already be installed manually on music so that the team can look at the work, but it should be off by default (check another site to confirm such as: `hs-sandbox-dev` or `islamicstudies-dev`. 
[ ] Once the module is installed, or if you're on the music site, confirm that there's a new button under 'Appearance' called `HumSci Mega Menu Settings`. Going there will turn on and off the mega menu. Feel free to turn it on / off and save to test.
[ ] With it on, confirm the mega menu is there and not seriously broken (but several people have already checked it out).

## Screenshots
![Screen Shot 2022-07-13 at 1 07 00 PM](https://user-images.githubusercontent.com/8405274/178812909-582039df-b6b0-406d-8be1-10985029a8a6.png)
